examples/vm_scheduling/rule_based_algorithm/first_fit.py

```diff
@@ -0,0 +1,29 @@
+from maro.simulator import Env
+from maro.simulator.scenarios.vm_scheduling.common import Action
+from maro.simulator.scenarios.vm_scheduling import AllocateAction, DecisionPayload, PostponeAction
+
+from algorithm import VMSchedulingAgent
+
+
+class FirstFit(VMSchedulingAgent):
+    def __init__(self):
+        super().__init__()
+        self._pm_list: list[int] = list()
```

Comment:
no need to maintain a list, since every time the newly appended element is `len(self._pm_list)`, keep a count is equal.